setup.cfg

```diff
@@ -39,16 +39,3 @@ ignore = E203, # Whitespace before ':' (black compatibility)
          B302, # this is a python 3 compatibility warning, not relevant since don't support python 2 anymore
 
 jobs=8
-
-# To be kept consistent with "Import Formatting" section in CONTRIBUTING.rst
-[tool:isort]
-multi_line_output=3
-include_trailing_comma=True
-force_grid_wrap=0
-combine_as_imports=True
-line_length=88
```

Comment:
I'm guessing these config settings are what are covered by the new `profile=black` setting, is that right?

Reply:
That's the whole idea of `profile="black"` yeah. To be completely transparent I didn't find a fully fleshed description of what's covered by it exactly, so I just removed everything that had no impact on `isort yt/`, and assumed it was simply redundant.

Reply:
It looks like it are these settings: https://black.readthedocs.io/en/stable/compatible_configs.html#configuration 

which are these lines plus a few more!

Reply:
cool !